site/documentation.markdown

```diff
@@ -52,7 +54,7 @@ Short, dense, classic ways to hit the ground running
 ## Online Resources
 
 Curated resources put together by Haskellers:
-
+ 
```

Comment:
Would you mind undoing this whitespace change? Not a big deal but personally I tend to prefer diffs that are as clear as possible.